project2/kb.json

```diff
@@ -1,4 +1,58 @@
 {
+  "meats": [
```

Comment:
`meats` and `meat-substitutes` were moved to `ingredients: protein: meats` and `ingredients: protein: vegetarian` below in a previous commit so this should probably be removed